kong/clustering/wrpc_control_plane.lua

```diff
@@ -167,8 +170,14 @@ function _M:export_deflated_reconfigure_payload()
   kong_dict:set(shm_key_name, cjson_encode(self.plugins_configured))
 
   local service = get_config_service(self)
+
+  -- yield between steps to provent long delay
+  local config_json = assert(cjson_encode(config_table))
+  yield()
```

Comment:
I think `yield` should be called before `cjson_encode` and `deflate_gzip`

Reply:
I am referring to the WebSocket version of DP/CP.